sw/device/lib/testing/test_rom/test_rom.c

```diff
@@ -113,6 +113,25 @@ bool rom_test_main(void) {
       mmio_region_from_addr(TOP_EARLGREY_FLASH_CTRL_CORE_BASE_ADDR)));
   CHECK_DIF_OK(dif_flash_ctrl_start_controller_init(&flash_ctrl));
   flash_ctrl_testutils_wait_for_init(&flash_ctrl);
+#if !OT_IS_ENGLISH_BREAKFAST
```

Comment:
Just curious, is test_rom share among all top profiles?

Reply:
currently yes. The whole multi-top strategy has yet to be implemented.